tests/test_task.py

```diff
@@ -864,6 +864,9 @@ def test_autoclassify_errors(autoclassification_config, expected_error, error_me
 ONE_FAILURE_TYPE_CRASH = {"group1": [("test1.js", FailureType.CRASH)]}
 ONE_FAILURE_TYPE_TIMEOUT = {"group1": [("test1.js", FailureType.TIMEOUT)]}
 ONE_FAILURE_TYPE_GENERIC = {"group1": [("test1.js", FailureType.GENERIC)]}
+TWO_FAILURES_SAME_TEST_TYPE_GENERIC = {
+    "group1": [("test1.js", FailureType.GENERIC), ("test1.js", FailureType.GENERIC)]
+}
```

Comment:
could you also make: TWO_FAILURES_SAME_TEST_TYPE_DIFFERENT?  Maybe a Timeout and Generic?

Reply:
Good idea, done in 517c502.